changelog/upgrade-notes.md

```diff
@@ -1,5 +1,7 @@
 ## Upgrade notes
 
+### v3.14.0
+
```

Comment:
Maybe it is not clear enough, but I think [that documentation](https://github.com/openlayers/ol3/wiki/Release-Procedure#create-a-changelog) is about creating a section for the current release.

> Now edit the changelog/upgrade-notes.md to add a new section at head of the file, ### v3.1.0 for example.

Since that doc is about creating a `v3.1.0` release, that just means to create a heading in the upgrade notes for the current release.